### ISSUE TITLE
FIXME: "compresslevel" in zipfile.ZipFile.__doc__ (for py < 3.7)

### ISSUE THREAD

**Issue by obfusk** [obfusk/apksigcopier#30]

**Comment by obfusk** [comment]
`zipfile.ZipFile.__init__.__code__.co_varnames`?

**Comment by eighthave** [comment]
how about just doing it like:
```python
def _write_to_zip(info_data, zf_out):
    if sys.version_info.major > 3 or sys.version_info.minor > 6:
        for info, data in info_data:
            zf_out.writestr(info, data, compresslevel=9)
    else:
        old = zipfile._get_compressor
        zipfile._get_compressor = lambda _: zlib.compressobj(9, 8, -15)
        try:
            for info, data in info_data:
                zf_out.writestr(info, data)
        finally:
            zipfile._get_compressor = old
```

**Comment by mimi89999** [comment]
@eighthave 
> `if sys.version_info.major > 3 or sys.version_info.minor > 6:`

I would say `if sys.version_info.major > 3 or (sys.version_info.major == 3 and sys.version_info.minor >= 7):`

**Comment by eighthave** [comment]
That's fine by me, but seems a bit pedantic since Python 2.7 is dead.

**Comment by obfusk** [comment]
> Python 2.7 is dead.

I don't think PyPy agrees :sweat_smile:

But I'm definitely not supporting it.

**Comment by obfusk** [comment]
fixed by b8d938b
